### DISCUSSION TITLE
zigbee2mqtt compatible relay for dry contacts

### DISCUSSION THREAD

**Discussion by acrossi** [Koenkk/zigbee2mqtt#6817] [Devices]
Howdy folks. Does anyone know if there is a zigbee2mqtt compatible relay to open /close a dry contact? I have a window opener / closer for a greenhouse that operates by way of two dry contacts (one to open, one to close). I see a bunch of light switches supported, but the only true relay I can see is the DIYRuZ DIYRuZ_R4_5, and I do not have the time to assemble the DIY components and make these units myself.

Or, alternatively, does anyone know where I can buy assembled DIYRuZ DIYRuZ_R4_5 units for shipment to the US? I don't speak russian!

Thanks for the ideas!

**Comment by Floris74** [comment]
Hey,

I had the same challenge. I bought this one 
![image](https://user-images.githubusercontent.com/81470608/112720182-2004fd00-8efd-11eb-913d-d11a396a256e.png)
The Moes MS-104Z. You can use it as an potential free relay

**Reply by acrossi** [reply]
Thanks! I will give it a try.

**Comment by kirovilya** [comment]
fresh device https://aliexpress.ru/item/1005002359014268.html
but already work with z2m

**Comment by kirovilya** [comment]
and with 2 relay https://aliexpress.ru/item/1005002347566295.html
